src/audio/mixer.c

```diff
@@ -186,8 +186,9 @@ static int mixer_params(struct comp_dev *dev,
 		return -EINVAL;
 	}
 
-	sinkb = list_first_item(&dev->bsink_list, struct comp_buffer,
-							source_list);
+	sinkb = list_first_item(&dev->bsink_list,
+				struct comp_buffer,
+				source_list);
 
```

Comment:
Just leave the first line untouched and align source_list under the parenthesis above.
Something like this. Lets hope github formatting works.
```
list_first_item(&dev->bsink_list, struct comp_buffer,
                          source_list);
```

Reply:
My apologies. I incorrectly created the pull request. Should have used a new branch for the second commit.
@dbaluta Would you like me to correct as you suggested and resend?

Reply:
@mh12gx2825 when you need to do a correction of a patch, just do it on the same branch. And us git push -f. This will override existing patch in this Pull Request and we will see it as you updated the patch with my comment.

Reply:
Okay. Followed your instruction and is looking good to me as you mentioned. Thank you.